web/src/components/navbar/NavBar.test.tsx

```diff
@@ -152,30 +249,107 @@ describe("<NavBar />", () => {
       useMockRoadmap({});
     });
 
-    const renderMobileTaskNav = (config?: { includeOperateRef: boolean }): RenderResult => {
-      setLargeScreen(false);
-
-      const operateRef: Record<string, OperateReference> = {
-        "some-tax-filing-identifier-1": {
-          name: "some-filing-name-1",
-          urlSlug: "some-urlSlug-1",
-          urlPath: "some-path",
-        },
+    const renderMobileTaskNav =
+      (isAuthenticated = IsAuthenticated.TRUE) =>
+      (config?: { includeOperateRef: boolean }): RenderResult => {
```

Comment:
can we rewrite this function-that-returns-a-function as something more readable?

Reply:
unfortunately no because of how displaysUserNameOrEmail uses it